src/pages/spawn/spawnHost/SpawnHostCard.stories.tsx

```diff
@@ -1,9 +1,7 @@
-import React from "react";
-import { MemoryRouter as Router } from "react-router-dom";
 import { SpawnHostCard } from "pages/spawn/spawnHost/SpawnHostCard";
 
 const host = {
-  expiration: "2020-08-21T18:00:07Z",
+  expiration: new Date("2020-08-21T18:00:07Z"),
```

Comment:
Is there a reason to change this from a string to an object? I think the SpawnHostCard was rendering correctly before.

Reply:
![image](https://user-images.githubusercontent.com/4605522/173911010-238640ca-9048-4876-8496-967ecec8485b.png)
Uptime and Expiration do expect a Date